### PR TITLE
octopus: PendingReleaseNotes: mgr/pg_autoscaler

### PR DESCRIPTION
**Pull request by kamoltat**
Added details describing the changes that
occured after merging #39248 into octopus
upstream.

Signed-off-by: Kamoltat <ksirivad@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,5 +1,11 @@
 15.2.8
```
Comment (neha-ojha) [inline PendingReleaseNotes:1]: should be 15.2.9 now

File: PendingReleaseNotes [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
 ------
+* mgr-pg_autoscaler: Autoscaler will now start out with scaling each
```
Comment (neha-ojha) [inline PendingReleaseNotes:3]: The autoscaler

File: PendingReleaseNotes [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+  pools to have a full complements of pgs from the start and will only
```
Comment (neha-ojha) [inline PendingReleaseNotes:4]: s/pools/pool/

File: PendingReleaseNotes [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+  decrease it when pools need more pgs due to increased usage. The solves
```
Comment (neha-ojha) [inline PendingReleaseNotes:5]: other pools

File: PendingReleaseNotes (hunk 2 above)
Comment (neha-ojha) [reply PendingReleaseNotes:3]: s/with/by/

File: PendingReleaseNotes (hunk 4 above)
Comment (neha-ojha) [reply PendingReleaseNotes:5]: How about reframing the last sentence? Maybe something like "This improves out of the box performance of Ceph by allowing more PGs to be created for a given pool."

**Comment by jdurgin** [comment]
looks good with neha's suggestions applied. we'll want the same release notes in master - can target to master and cherry-pick from there to octopus/nautilus

**Comment by kamoltat** [comment]
@neha-ojha thank you for proofreading this 👍

**Comment by kamoltat** [comment]
> looks good with neha's suggestions applied. we'll want the same release notes in master - can target to master and cherry-pick from there to octopus/nautilus/pacific

will create another PR for master and cherry-pick to pacific, nautilus

**Review by jdurgin — APPROVED** [review]

**Pull request merged**